crates/ruff/src/settings/mod.rs

```diff
@@ -293,15 +293,13 @@ impl Settings {
         })
     }
 
-    #[cfg(test)]
     pub fn for_rule(rule_code: Rule) -> Self {
         Self {
             rules: RuleTable::from_iter([rule_code]),
             ..Self::default()
         }
     }
 
-    #[cfg(test)]
```

Comment:
Why are these no longer `#[cfg(test)]`? Are they used outside of tests?

Reply:
Oh, perhaps for the configuration tests.

Reply:
they are used across crates now (the test has to live in ruff_cli)) and `cfg(test)` doesn't get passed across crate boundaries